go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/aquasecurity/table v1.8.0
 	github.com/aquasecurity/testdocker v0.0.0-20240730042311-4642e94c7fc8
 	github.com/aquasecurity/tml v0.6.1
-	github.com/aquasecurity/trivy-checks v1.8.1
+	github.com/aquasecurity/trivy-checks v1.8.2-0.20250403212859-379320b8673d
```

Comment:
note: use 1.8.2 once all the checks passed on this PR and we have a release for checks.